nexus/src/app/background/networking.rs

```diff
@@ -80,7 +80,11 @@ pub(crate) fn api_to_dpd_port_settings(
             LinkSettings {
                 params: LinkCreate {
                     autoneg: l.autoneg,
-                    lane: Some(LinkId(0)),
+                    // TODO: dendrite currently does nothing with this field.
+                    // Setting it to a value causes RPW reconciler passes to do
+                    // unnecessarily work, trying to set a field that dendrite
+                    // will not apply.
+                    lane: None,
```

Comment:
@rcgoodfellow Please sanity check me here, because `dendrite` _responds_ with `None` for this field whenever we check the config it has. I've started digging through dendrite's code to verify if we're really not using this field, or we're simply not sending the correct value back when we check the config.

Reply:
This does not seem right. Dendrite should be returning the lane assignment.

Reply:
Is it possibly due to this block?

https://github.com/oxidecomputer/dendrite/blob/41ddeab9d43d90a51e6fc1c236dc9982fc76f922/dpd/src/api_server.rs#L2761-L2770